### DISCUSSION TITLE
Ignore RepeatedResourceWarning

### DISCUSSION THREAD

**Discussion by caseyscarborough** [argoproj/argo-cd#8927] [Q&A]
Is it possible to ignore/whitelist specific warnings as a non-issue, such as the RepeatedResourceWarning? We have a case where sometimes a RepeatedResourceWarning occurs but it's really not an issue and we would like the warning to be removed from the UI.

I've tried searching around but can't seem to find much on it except maybe [this issue (#2115)](https://github.com/argoproj/argo-cd/issues/2115).

**Comment by KalenWessel** [comment]
I know this isn't an answer but I am also interested in the ability to ignore the `RepeatedResourceWarning ` message, ideally on a case-by-case basis. This way files I know will show up can be ignored while new unexpected duplicates would trigger the warning.

**Comment by rsavage-nozominetworks** [comment]
I am also interested in discovering how to silence this message.   According to the ArgoCD documentation, ArgoCD allows for an application to be sourced from multiple repos.   If there is a duplicate resource, the last definition overrides.   This is exactly what I want and need.   However, I don't need the warning message because it's intended.

**Comment by noah-hein** [comment]
Interestingly enough the ArgoCD rbac page promotes creating multiple argocd-rbac-cm ConfigMaps to make it easier to manage the policy csv, which ends up throwing this warning.

**Comment by dannysauer** [comment]
Looking at the code which detects dups:
https://github.com/argoproj/argo-cd/blob/e5417e1eb383fa72f00f7a18bd29af3047ce4bf4/controller/state.go#L422-L432
...it appears there is no test for a configuration option to silence the specific warning.

Though it should be entirely possible to add a configuration in the app which _does_ suppress the warning.  Maybe a list of resource_type/name strings?

**Comment by Bak3y** [comment]
Casting another vote for the ability to ignore warnings here.

My specific use case is I'm using a using a wrapper chart with multiple instances of the external-dns subchart (a common pattern for split DNS), the CRDs are identical and deduplicated correctly, ServerSideApply=true handles them fine, and the warning is pure noise with no way to suppress it.

EDIT - https://github.com/argoproj/argo-cd/issues/28026
